src/winetricks

```diff
@@ -15275,6 +15277,18 @@ load_ie8()
             w_try mv "${W_SYSTEM32_DLLS}"/${dll}.dll "${W_SYSTEM32_DLLS}"/${dll}.dll.bak
     done
 
+    if [ "${W_ARCH}" = "win64" ]; then
+        if [ -f "${W_PROGRAMS_UNIX}/Internet Explorer/iexplore.exe" ]; then
+            w_try mv "${W_PROGRAMS_UNIX}/Internet Explorer/iexplore.exe" "${W_PROGRAMS_UNIX}/Internet Explorer/iexplore.exe.bak"
+        fi
+
+        for dll in browseui inseng itircl itss jscript msctf mshtml shdoclc shdocvw shlwapi urlmon; do
```

Comment:
The diff doesn't seem to show properly in github, but IMO this list of dlls shouldn't be duplicated from https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L15272.

Instead add the W_ARCH check there, and if on 64-bit, also move the 64-bit dlls away.

Reply:
Done.